lib/screens/create_wallet/generate_mnemonic_card.dart

```diff
@@ -57,21 +57,21 @@ class GenerateMnemonicCardState extends State<GenerateMnemonicCard>
       children: [
         Text(
           localization.generateMnemonic01(_radioWordCount),
-          style: theme.textTheme.bodyLarge,
+          style: theme.textTheme.bodyMedium,
```

Comment:
The mnemonic fontSize is too small. Also the description text seems a bit too small too.